HopsanGUI/HcomHandler.cpp

```diff
@@ -363,7 +363,10 @@ HcomHandler::HcomHandler(TerminalConsole *pConsole) : QObject(pConsole)
     registerInternalFunction("lp1", "Applies low-pass filter of first degree to vector","Usage: lp1(vector, frequency)\nUsage: lp1(vector, timevector, frequency)");
     registerInternalFunction("ddt", "Differentiates vector with respect to time (or to custom vector)","Usage: ddt(vector)\nUsage: ddt(vector, timevector)");
     registerInternalFunction("int", "Integrates vector with respect to time (or to custom vector)", "Usage: int(vector)\nUsage: int(vector, timevector)");
-    registerInternalFunction("fft", "Generates frequency spectrum plot from vector","Usage: fft(vector)\nUsage: fft(vector, power[true/false])\nUsage: fft(vector, timevector)\nUsage: fft(vector, timevector, power[true/false])");
+    registerInternalFunction("esd", "Generates energy spectral density from vector","Usage: esd(vector, timevector [optional], windowing[rectangular/flattop/hann, optional], mintime[optional], maxtime[optional])\n");
+    registerInternalFunction("psd", "Generates power spectral density from vector","Usage: psd(vector, timevector [optional], windowing[rectangular/flattop/hann, optional], mintime[optional], maxtime[optional])\n");
```

Comment:
Is this a new way of noting optional parameters, before I think we did [name] to indicate optional. I have not tested this branch to see if this looks better though

Reply:
No the old way is better. This is too verbose. I just didn't notice, since no other internal function have any optional arguments.